src/gmail.d.ts

```diff
@@ -863,7 +863,7 @@ interface GmailObserve {
       trigger the related event and fire off any relevant bound callbacks
```

Comment:
Register could be changed to:
```ts
register(action: T, args: string | StringDict): void;
````
but that would be a breaking change

Reply:
Updated slightly given recent changes:
```ts
register(action: T, args: string | DomObserverConfig): void;
```
which would limit the values of action to those that are pre-declared.